setup.py

```diff
@@ -0,0 +1,44 @@
+from setuptools import setup, find_packages, find_namespace_packages
+import pathlib
+
+here = pathlib.Path(__file__).parent.resolve()
+
+# workaround a bug introduced by pyproject.toml
+# https://github.com/pypa/pip/issues/7953#issuecomment-645133255
+import site, sys; site.ENABLE_USER_SITE = True
+
+setup(
+  name='spinalcordtoolbox-data-<dataset>',
+  description='Part of https://github.com/neuropoly/spinalcordtoolbox',
+  long_description=(here / 'README.md').read_text(encoding='utf-8'),
+  long_description_content_type='text/markdown',
+  author='Neuropoly',
+  author_email='neuropoly@googlegroups.com',
+  url='https://spinalcordtoolbox.com/',
+  project_urls={
+      'Source': 'https://github.com/sct-data/<dataset>',
+      #'Documentation': '',
```

Comment:
Could we maybe point this to `spinalcordtoolbox.com` as a sort of blanket link for any future documentation on our datasets?

Reply:
Nice that's the sort of feedback I was hoping for